script/tool/lib/src/common/git_version_finder.dart

```diff
@@ -28,15 +28,6 @@ class GitVersionFinder {
   /// The base branche used to find a merge point if baseSha is not provided.
   final String _baseBranch;
 
-  static bool _isPubspec(String file) {
-    return file.trim().endsWith('pubspec.yaml');
-  }
-
-  /// Get a list of all the pubspec.yaml file that is changed.
-  Future<List<String>> getChangedPubSpecs() async {
-    return (await getChangedFiles()).where(_isPubspec).toList();
-  }
```

Comment:
This code was only used in one place (and isn't really related to `git` so it's not clear why it was in this class), so I inlined it where it was used so that I could use the cached changed files list instead of recomputing it here.